Sources/VLCRendererDiscoverer.m

```diff
@@ -39,8 +40,8 @@ static void HandleRendererDiscovererItemAdded(const libvlc_event_t *event, void
     @autoreleasepool {
         [[VLCEventManager sharedManager] callOnMainThreadObject:(__bridge id)(self)
                                                      withMethod:@selector(itemAdded:)
-                                           withArgumentAsObject:[[VLCRendererItem alloc] initWithCItem:
-                                                                 event->u.renderer_discoverer_item_added.item]];
+                                           withArgumentAsObject:[NSValue valueWithPointer:
+                                                                 event->u.renderer_discoverer_item_added.item]] ;
```

Comment:
what is this u btw? That's not very discriptive

Reply:
If I'm not mistaken, it's a union defined in libvlc.